### ISSUE TITLE
34th element breaks sequence of elements. 

### ISSUE THREAD

**Issue by ilandblack** [nggepe/simple_grid#17]
34th element breaks sequence of elements. 12th becomes 1st, 34th -> 22

Flutter 3.24.5
Dart 3.5.4
simple_grid 0.1.4

![image](https://github.com/user-attachments/assets/802681b4-02e6-49b6-b748-3e14bfca9594)

Code to reproduce:

```
class Test extends StatelessWidget {
  const Test ({
    Key? key,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: SingleChildScrollView(
        physics: const AlwaysScrollableScrollPhysics(),
        child: Padding(
          padding: const EdgeInsets.all(kDefaultPadding),
          child: SpGrid(
            width: MediaQuery.of(context).size.width,
            spacing: kDefaultPadding / 2,
            runSpacing: kDefaultPadding,
            children: [
              for (int i = 1; i <= 34; i++)
                SpGridItem(
                  sm: 12,
                  lg: 4,
                  child: Container(
                    height: 50,
                    color: Color(0xFF000000),
                    child: Center(child: Text('$i')),
                  ),
                ),
            ],
          ),
        ),
      ),
    );
  }
}
```
BTW Thanks for the lib. Best grid i found.

**Comment by nggepe** [comment]
thank you for reporting this issue. I'll check it soon

**Comment by nggepe** [comment]
Hi @ilandblack , sorry I just checked it. Yes, you're right, I think it is because the order. I'll try to fix it as soon as possible.

<img width="1198" alt="Screenshot 2024-12-29 at 18 25 29" src="https://github.com/user-attachments/assets/fbc939a6-769e-4960-ae4b-2b80bd79f84f" />

**Comment by nggepe** [comment]
hi @ilandblack I already fixed this issue with version 0.1.5

**Comment by nggepe** [comment]
If still there is an issue like this on your code, you can reopen this issue.
